pkg/api/schema/schema.proto

```diff
@@ -321,6 +321,13 @@ message VerifiableGetRequest {
 	uint64 proveSinceTx = 2;
 }
 
+message ServerInfoRequest {}
+
+message ServerInfoResponse {
+	bool status = 1;
```

Comment:
Please add some comments what the fields in the `ServerInfoResponse` message mean.